currencyii/src/main/java/nl/tudelft/trustchain/currencyii/ui/bitcoin/SharedWalletTransaction.kt

```diff
@@ -62,7 +62,7 @@ class SharedWalletTransaction : BaseFragment(R.layout.fragment_shared_wallet_tra
     }
 
     private fun transferFundsClicked() {
-        if (!validateCreationInput()) {
+        if (!validateTransferInput()) {
             activity?.runOnUiThread {
                 alert_view.text = "Failed: Bitcoin PK should be a string, minimal satoshi amount: 5000"
```

Comment:
Use the (not yet present) min satoshi constant here to display the correct min. satoshi amount

Reply:
Will do

Reply:
Fixed